MediaBrowser.Model/Providers/ExternalIdInfo.cs

```diff
@@ -16,6 +16,12 @@ public class ExternalIdInfo
         /// <value>The key.</value>
         public string Key { get; set; }
 
+        /// <summary>
+        /// Gets or sets the type.
```

Comment:
This needs to be more descriptive. If you can, the documentation for the `Key` and `Name` properties should also be updated to describe what they are actually used for.

Reply:
done